### PR TITLE
Put up final demand artifact

### PR DESCRIPTION
**Pull request by bamader**
## Background
Fast follows on the demand estimation work that puts together everything into one CSV.

## Additional Context
No extra context, I don't think--everything we talked about and agreed on is here! Kyle, I added the county sub  code to the aggregation cuz I  know you wanted to use  it for census joins.

### CODE REVIEW

File: src/demand_estimation/calculate_town_demand.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+
+BASE_TOWN_FILE = "town_data.csv"
+CALCULATED_PARENT_METRIC_FILE = "town_demand_estimated_data.csv"
+SMI_ELIGIBLE_CHILDREN_FILE = "town_estimates.csv"
```
Comment (kmagida110) [inline src/demand_estimation/calculate_town_demand.py:5]: Did you move this file here?

Comment (kmagida110) [reply src/demand_estimation/calculate_town_demand.py:5]: If not, can you?

File: src/demand_estimation/calculate_town_demand.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+OUT_FILE = "final_demand_by_town.csv"
+
+towns = pd.read_csv(BASE_TOWN_FILE)
+towns.drop(columns=[c for c in towns.columns if not c in ['NAME', 'COUSUBFP']], inplace=True)
```
Comment (kmagida110) [inline src/demand_estimation/calculate_town_demand.py:9]: Thanks for the county sub! Can you leave the county too, I'm not sure if the sub is unique across the state

File: src/demand_estimation/calculate_town_demand.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+towns.rename(columns={'NAME': 'town'}, inplace=True)
+parent_metric_df = pd.read_csv(CALCULATED_PARENT_METRIC_FILE)
+smi_df = pd.read_csv(SMI_ELIGIBLE_CHILDREN_FILE)
+df = towns.merge(parent_metric_df, on='town')
+df = df.merge(smi_df, on='town')
+df.drop(columns=['min_full_estimate', 'max_full_estimate'], inplace=True)
```
Comment (kmagida110) [inline src/demand_estimation/calculate_town_demand.py:15]: 👍

**Review by kmagida110 — APPROVED** [review]
LGTM as long as the files are all in the right places.

**Review by imcurtis — APPROVED** [review]

File: src/demand_estimation/calculate_town_demand.py (hunk 1 above)
Comment (bamader) [reply src/demand_estimation/calculate_town_demand.py:5]: Yup can do!

File: src/demand_estimation/calculate_town_demand.py (hunk 2 above)
Comment (bamader) [reply src/demand_estimation/calculate_town_demand.py:9]: Yeah sure thing

**Pull request merged**